example.py

```diff
@@ -28,13 +30,15 @@ def run_example():
     # Get credentials
     credentials = get_key_vault_credentials()
 
-    # Create a KeyVault client
+    # Deprecated Libraries, create a KeyVault client 
```

Comment:
Is there a reason for the "deprecated libraries" comments?

Reply:
@mccoyp - Added "Deprecated Library" to indicate that it is code for track1 (old SDK) rather than track2 (latest SDK).

Reply:
I see -- I missed the note about this targeting an older version of the SDK. Everything looks good then! We may want to update this sample in the future to follow a newer version though, since it's pretty far behind.